docs/modules/ROOT/pages/autotasks.adoc

```diff
@@ -288,19 +288,20 @@ Autotasks are executed in a https://nodejs.org/dist/latest-v16.x/docs/api/[node
 "@datadog/datadog-api-client": "^1.0.0-beta.5",
 "@gnosis.pm/safe-core-sdk": "^0.3.1",
 "@gnosis.pm/safe-ethers-adapters": "^0.1.0-alpha.3",
-"axios": "^0.27.2",
-"axios-retry": "3.1.9",
-"defender-admin-client": "1.37.0-rc.2",
-"defender-autotask-client": "1.37.0-rc.2",
+"axios": "^1.4.0",
+"axios-retry": "3.5.0",
+"defender-admin-client": "1.44.0",
+"defender-autotask-client": "1.44.0",
 "defender-autotask-utils": "1.34.0",
-"defender-kvstore-client": "1.37.0-rc.2",
-"defender-relay-client": "1.37.0-rc.2",
-"defender-sentinel-client": "1.37.0-rc.2",
+"defender-kvstore-client": "1.44.0",
+"defender-relay-client": "1.44.0",
+"defender-sentinel-client": "1.44.0",
```

Comment:
should we change this to use `@openzeppelin/defender-*-client` with latest version instead

Reply:
Good question.

We would need to update stacks/layers/autotasks-runtime/nodejs/package.json to use those newer packages first.

That would require from us more attention as we need docs for users on how to migrate. 

I'll create ticket.

Reply:
We have ticket already
https://linear.app/openzeppelin-development/issue/PLAT-1697/reference-openzeppelin-scoped-defender-client-packages-in-autotask